app/src/main/res/values/styles_v2.xml

```diff
@@ -739,6 +739,24 @@
     <item name="iconTint">?attr/colorOnSecondary</item>
   </style>
 
+  <style name="Clinic.V2.MaterialButton.Blue2">
+    <item name="backgroundTint">@color/blue2</item>
+    <item name="android:textColor">?attr/colorSecondary</item>
+    <item name="iconTint">?attr/colorSecondary</item>
```

Comment:
I am not entirely sure of this mix of attrs and explicit colours. Can we use explicit colours instead?